wiki/Past_osu!store_items/de.md

```diff
@@ -0,0 +1,125 @@
+---
+tags:
+  - store
+  - merchandise
+  - legacy
+  - Shop
+---
+
+# Vergangene Artikel aus dem osu!store
+
+Auf dieser Seite werden verschiedene Produkte gezeigt, die mal im [osu!store](https://osu.ppy.sh/store/listing) verfügbar waren, mittlerweile aber nicht mehr produziert werden und auch nicht zum Verkauf stehen.
+
+## osu! goods
+
+![Storeseite von osu! goods CafePress](img/osu-goods.png)
+
+**osu! goods** war eine Kollektion von offiziellen osu!-Markenartikeln, die über den Onlinehändler [CafePress](https://en.wikipedia.org/wiki/CafePress) zwischen 2009 und 2012 verkauft wurden. Alle Artikel wurden von CafePress gedruckt und produziert, wobei ein Teil der Einnahmen in die [Spielentwicklung](https://osu.ppy.sh/store/listing) flossen.
```

Comment:
The link for "supporting the game" is a bit unfortunate in that position

Maybe

```suggestion
**osu! goods** war eine Kollektion von offiziellen osu!-Markenartikeln, die über den Onlinehändler [CafePress](https://en.wikipedia.org/wiki/CafePress) zwischen 2009 und 2012 verkauft wurden. Alle Artikel wurden von CafePress gedruckt und produziert, wobei ein Teil der Einnahmen [in die Spielentwicklung flossen](https://osu.ppy.sh/store/listing).
```

or alternatively

```suggestion
**osu! goods** war eine Kollektion von offiziellen osu!-Markenartikeln, die über den Onlinehändler [CafePress](https://en.wikipedia.org/wiki/CafePress) zwischen 2009 und 2012 verkauft wurden. Alle Artikel wurden von CafePress gedruckt und produziert. Ein Teil der Einnahmen wurde zur [Unterstützung der Spielentwicklung](https://osu.ppy.sh/store/listing) verwendet.
```